include/h2o/rand.h

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2014 DeNA Co., Ltd.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to
+ * deal in the Software without restriction, including without limitation the
+ * rights to use, copy, modify, merge, publish, distribute, sublicense, and/or
+ * sell copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+ * FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
+ * IN THE SOFTWARE.
+ */
+#ifndef h2o__rand_h
+#define h2o__rand_h
+#include <sys/param.h>
+
+#ifdef  BSD
+#if OpenBSD >= 201405 || __NetBSD_vVersion__ > 700000000
+#define h2o_srand()
+#define h2o_rand()      \
+    arc4random()
+#endif
+#else
+#define h2o_srand()     \
+    srand(time(NULL))
```

Comment:
Could you please change the seed to `time(NULL) ^ getpid()` (and that means that we should include unistd.h as well)?

The reason I suggest this is because the ticket updater defined in src/ssl.c uses `sleep(rand())` to avoid collision between the server processes, and therefore it is desirable to have different random number sequence for servers that get spawned at the same moment.